java-frontend/src/main/java/org/sonar/java/se/constraint/ConstraintManager.java

```diff
@@ -187,5 +188,8 @@ public boolean isNull(ProgramState ps, SymbolicValue val) {
     List<ProgramState> trueConstraint = sv.setConstraint(unstack.state, BooleanConstraint.TRUE);
     return new Pair<>(falseConstraint, trueConstraint);
   }
-  
+
+  public SymbolicValue createSymbolicValue(BytecodeCFGBuilder.Instruction inst) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "inst". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)